WebContent/js/components/JobInstance.js

```diff
@@ -95,14 +95,18 @@ class JobInstance extends React.Component {
         const statusStyleActive = { display: 'inline' };
         const statusStyleAbend = { color: 'red', display: 'inline' };
         const statusStyleComplete = { color: 'grey', display: 'inline' };
-        if (job.get('returnCode')) {
-            if (job.get('returnCode').toLowerCase().includes('abend') || job.get('returnCode').toLowerCase().includes('jcl error')) {
-                return (<div style={statusStyleAbend}> [{job.get('returnCode')}]</div>);
+        if (job.get('status').toLowerCase().includes('output')) {
+            if (job.get('returnCode')) {
+                if (job.get('returnCode').toLowerCase().includes('abend') || job.get('returnCode').toLowerCase().includes('jcl error')
+                    || job.get('returnCode').toLowerCase().includes('sys fail') || job.get('returnCode').toLowerCase().includes('conv error')) {
+                    return (<div style={statusStyleAbend}> [{job.get('returnCode')}]</div>);
+                }
+                if (job.get('returnCode').toLowerCase().includes('cc') || job.get('returnCode').toLowerCase().includes('canceled')) {
+                    return (<div style={statusStyleComplete}> [{job.get('returnCode')}]</div>);
+                }
+                return (<div style={statusStyleActive}> [{job.get('returnCode')}]</div>);
```

Comment:
I'd also like to see a check here that the job is in active state before setting the active style, so `if job.get('status').includes('active')`

Reply:
Could you explain more here? 
I am not sure where would you like to add a check for active status.

Before my commit, When  job state has returnCode and returnCode does not include one of "abend", "jcl error", "cc", or "canceled" the function will set active style as default. 

Not sure why render job to active style here when error code is none of known 4 value before, so I just remain this here. 



Thanks.

Reply:
I made a mistake the first time I read this, however there still needs to be a change.

The line: `return (<div style={statusStyleActive}> [{job.get('returnCode')}]</div>);` is inside an if block that requires the job to have a return code, if a job has a return code it should not be displayed as active. Therefore I think this line should be removed as it should never be needed.

Reply:
Yes, I agree with you, if job has a return code, we should not render job as active style.

I think if a job has a return code and return code is none of we have covered ('abend', 'jcl error', 'cc', 'canceled', 'sys fail', 'conv error’, 'sec error'), we still need to give a default render style, what do you think?

May be the default style can be complete style.

Reply:
Yes, default can be statusStyleComplete